### DISCUSSION TITLE
Translation not working for boolean

### DISCUSSION THREAD

**Discussion by carolin-skiply** [directus/directus#13812] [Archive]
Hi guys,

I am trying to use translation with a boolean but does not seem to work. I probably miss something

Here is my settings:
![image](https://user-images.githubusercontent.com/34483024/172882146-d99f5ad7-823d-4288-aff3-bc4a981fd7aa.png)

Here is the result:
![image](https://user-images.githubusercontent.com/34483024/172882036-ee88fec8-5a4d-47f5-b078-48bba74fa12e.png)

Thanks for your help!

**Comment by carolin-skiply** [comment]
Hi guys,
Any idea on what I am doing wrong here ?

Thanks for the help,
Caroline.

**Reply by azrikahar** [reply]
Nothing wrong with what you did, but more so it's not working as expected! This was tracked in #13814 and has since been resolved by #13861 👍 The fix will be included in the next release.

**Reply by carolin-skiply** [reply]
Alright! 
Thank you for getting back to me.
